content/en/opentelemetry/otlp_endpoint.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Datadog OTLP Trace Intake Endpoint
+private: true
+further_reading:
+  - link: "https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/"
+    tag: "External"
+    text: "General OpenTelemetry SDK Configuration"
+  - link: "https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/"
+    tag: "External"
+    text: "OpenTelemetry Environment Variable Spec"
+  - link: "https://opentelemetry.io/docs/reference/specification/protocol/exporter/ "
+    tag: "External"
+    text: "OpenTelemetry Protocol Exporter"
+---
+{{< site-region region="ap1,gov" >}}
+<div class="alert alert-danger">This feature is not available for the {{< region-param key=dd_datacenter code="true" >}} site.</div>
+{{< /site-region >}}
+
+## Overview
+
+Datadog's OpenTelemetry protocol (OTLP) intake API endpoint allows you to send traces directly to Datadog, without running the [Datadog Agent][2] or [OpenTelemetry Collector + Datadog Exporter][1].
+
+You might prefer this option if you don't want the overhead of using the Datadog Agent or OpenTelemetry Collector.
+
+## Configuration
+
+To export OTLP data to the Datadog OTLP intake endpoint:
+
+1. [Configure the OTLP HTTP Protobuf exporter](#configure-the-exporter).
+   - Set the Datadog OTLP intake endpoint.
+   - Configure the required HTTP headers.
+1. (Optional) [Set `dd-otel-span-mapping` HTTP header](#optional-map-or-filter-span-names) to map or filter spans.
+
+### Configure the exporter
+
+To send OTLP data to the Datadog OTLP intake endpoint, you need to use the OTLP HTTP Protobuf exporter.
+
+#### Automatic instrumentation
+
+If you are using [OpenTelemetry automatic instrumentation][3], set the following environment variables:
```

Comment:
```suggestion
If you are using [OpenTelemetry automatic instrumentation][3], set the following environment variables, replacing `{DD_API_KEY}` with your Datadog API Key and `{YOUR_ORG]` with your organization:
```

I'm not sure what `YOUR_ORG` is supposed to be, so this might need to be changed a bit.